scipy/stats/_multivariate.py

```diff
@@ -3850,3 +3851,431 @@ def rvs(self, dim, size=1, random_state=None):
+            Probability density function evaluated at `x`
+
+        Notes
+        -----
+        %(_doc_callparams_note)s
+        """
+        out = np.exp(self.logpmf(x, m, n))
+        return out
+
+    def mean(self, m, n):
+        """
+        Mean of the Multivariate Hypergeometric distribution
+
+        Parameters
+        ----------
+        %(_doc_default_callparams)s
+
+        Returns
+        -------
+        mean : array_like or scalar
+            The mean of the distribution
+        """
+        M, m, n, mncond = self._process_parameters(m, n)
+        M, n = M[..., np.newaxis], n[..., np.newaxis]
+        mu = n*(m/M)
+        return self._checkresult(mu, mncond, np.NAN)
+
+    def var(self, m, n):
+        """
+        Variance of the Multivariate Hypergeometric distribution.
+
+        Parameters
+        ----------
+        %(_doc_default_callparams)s
+
+        Returns
+        -------
+        cov : array_like
+            The covariance matrix of the distribution
+        """
+        M, m, n, mncond = self._process_parameters(m, n)
+        M, n = M[..., np.newaxis], n[..., np.newaxis]
+        output = n * m/M * (M-m)/M * (M-n)/(M-1)
+        return self._checkresult(output, mncond, np.NAN)
+
+    def cov(self, m, n):
+        """
+        Covariance matrix of the Multivariate Hypergeometric distribution.
+
+        Parameters
+        ----------
+        %(_doc_default_callparams)s
+
+        Returns
+        -------
+        cov : array_like
+            The covariance matrix of the distribution
+        """
+        # cov( x_i,x_j ) = -n * (M-n)/(M-1) * (K_i*K_j) / (M**2)
+        M, m, n, mncond = self._process_parameters(m, n)
+        M = M[..., np.newaxis, np.newaxis]
+        n = n[..., np.newaxis, np.newaxis]
+        output = (-n * (M-n)/(M-1) / (M**2) *
+                  np.einsum("...i,...j->...ij", m, m))
+        dim = m.shape[-1]
+        M, n = M[..., 0, 0], n[..., 0, 0]
+        # diagonal entries need to be computed differently
+        for i in range(dim):
+            output[..., i, i] = (n * (M-n)/(M-1) *
+                                 m[..., i]*(M-m[..., i]) / (M**2))
+
+        return self._checkresult(output, mncond, np.NAN)
+
+    def rvs(self, m, n, size=1, random_state=None):
+        """
+        Draw random samples from a Multivariate Hypergeometric distribution.
+
+        Parameters
+        ----------
+        %(_doc_default_callparams)s
+        size : integer or iterable of integers, optional
+            Number of samples to draw (default 1).
+        %(_doc_random_state)s
+
+        Returns
+        -------
+        rvs : array_like
+            Random variates of shape (`size`, `len(p)`)
+
+        Notes
+        -----
+        %(_doc_callparams_note)s
+        """
+        M, m, n, mncond = self._process_parameters(m, n)
+
+        random_state = self._get_random_state(random_state)
+        try:
+            # Generator is only available in numpy >= 1.17
```

Comment:
![image](https://user-images.githubusercontent.com/6570539/93001813-89d03300-f4e6-11ea-88dc-82d1fe2b275d.png)

Reply:
tested in `test_rvs_numpy`!